bin/adam-pyspark

```diff
@@ -23,10 +23,6 @@ ADAM_REPO="$(cd `dirname $0`/..; pwd)"
 CLASSPATH=$("$ADAM_REPO"/bin/compute-adam-classpath.sh)
 ADAM_JARS=$("$ADAM_REPO"/bin/compute-adam-jars.sh)
 
-if [ ! -z "$ADDL_JARS" ]; then
-  ADAM_JARS="$ADAM_JARS","$ADDL_JARS"
-fi
```

Comment:
I am OK with removing `ADDL_JARS`.

As an aside, how will options be set? Do they need to preface the arguments they are passing to ADAM on the command line with any additional `--conf` flags?

E.g., does:
```
cd $ADAM_HOME
export ADAM_OPTS="--conf spark.halt-and-catch-fire=false"
./bin/adam-shell transform myBamFile.bam myAdamFile.adam
```

become:

```
cd $ADAM_HOME
./bin/adam-shell --conf spark.halt-and-catch-fire=false transform myBamFile.bam myAdamFile.adam
```